cmd/v/internal/help/default.txt

```diff
@@ -10,7 +10,8 @@ Examples:
 
 The commands are:
    build             Build V code in the provided path (default).
-   create            Setup the file structure for a V project.
+   new               Setup the file structure for a V project.
```

Comment:
I think this will be better:
`Setup the file structure for a V project (in a sub folder).`

Most people will not read the comments in the source, while I hope they will read the `v help` . It is worth mentioning the fact that it will create a sub folder here.

Reply:
I did a `git commit --amend` for this and force pushed. Resolved.